packages/@react-native-windows/cli/src/config/configUtils.ts

```diff
@@ -83,7 +83,7 @@ export function findSolutionFiles(winFolder: string): string[] {
     return [allSolutions[0]];
   }
 
-  var solutionFiles = [];
```

Comment:
Unlike pretty much any other language `var`'s access is scoped to the entire function instead of the lexical scope where it is declared in. That's why ES6 replaced it with `let` and `const` and it shouldn't be used.